exonum/src/api/error.rs

```diff
@@ -63,3 +69,23 @@ impl From<failure::Error> for Error {
         Error::InternalError(e)
     }
 }
+
+pub(crate) fn convert_error(
+    error: JsonPayloadError,
+    max_payload_len: usize,
+    content_len: String,
+) -> Error {
+    match error {
+        JsonPayloadError::Overflow => Error::PayloadTooLarge(format!(
+            "Allowed payload length is: {}, but try to send: {}",
```

Comment:
I would suggest `but it was attepted to send` or `but got` (same in one other place).